src/ExposureHistory/ExposureList.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FunctionComponent } from "react"
+import { View } from "react-native"
+import { useExposureContext } from "../ExposureContext"
+import ExposureListItem from "./ExposureListItem"
+
+const ExposureList: FunctionComponent = () => {
```

Comment:
Could this component fit in the same file we have the `ExposureHistory`? Or perhaps since this one and the list item are entirely dependant on the History component we could have an inner folder to contain them?

Reply:
Yeah this one turns out not to be doing much aside from mapping, I think it can be part of the History component!